ci/push_docker.sh

```diff
@@ -23,14 +22,27 @@ function release() {
             return 1
         fi
         tag_exact="${DOCKER_REPO}:${weaviate_version}"
+  else
+    pr_title="$(echo -n "$PR_TITLE" | tr '[:upper:]' '[:lower:]' | tr -c -s '[:alnum:]' '-')"
+    tag_preview="${DOCKER_REPO}:preview-${pr_title}-${git_hash}"
   fi
 
-  args=("--build-arg=GITHASH=$git_hash" "--platform=linux/amd64,linux/arm64" "--target=weaviate" "-t=$tag_latest" "--push")
+  # args=("--build-arg=GITHASH=$git_hash" "--platform=linux/amd64,linux/arm64" "--target=weaviate" "-t=$tag_latest" "--push")
```

Comment:
reenable

Reply:
Haha, sorry, I put the status to "Draft" again as I started implementing Andrzej's review feedback. will fix this before marking it as ready again.